db/migration/V1_0__create_shop_schema.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA IF NOT EXISTS shop
```

Comment:
for future reference, this db.migration folder usually lives in the "resources" folder 📁